oidc-client-app-samples/ios-client-app-sample/Pods/AppAuth/Source/OIDClientMetadataParameters.h

```diff
@@ -0,0 +1,51 @@
+/*! @file OIDClientMetadataParameters.h
+    @brief AppAuth iOS SDK
+    @copyright
+        Copyright 2016 The AppAuth for iOS Authors. All Rights Reserved.
```

Comment:
Shall we change the year to 2018

Reply:
This is actually a file from the AppAuth for iOS library. The reason for committing dependency binaries is because we are using a specific branch of the AppAuth library. These files can be removed once the branch gets merged.